docs/reference/remap2/features/stateless.cue

```diff
@@ -0,0 +1,9 @@
+remap2: features: stateless: {
+	title:       "Stateless"
+	description: """
+		VRL is stateless, operating on a single Vector event at a time. This limits the complexity of VRL programs,
```

Comment:
Personally, I prefer "script" to "program." This applies not just here but throughout the VRL extended universe. When I think of a script I think of something that runs once from beginning to end; when I think of a program I think of something potentially long running and more complex. The term program just seems *heavier* somehow. Purely subjective.

Reply:
cc @JeanMertz. I'm indifferent.

Reply:
I prefer the term "program", for two reasons:

1. A program signifies something "more" than a script. A script is a file with a bunch of text that's executed when you run it. Whereas a program is something that's pre-compiled/prepared before you run it. This fits better with VRL, since we do a lot of work on your source to validate correctness before you run it. Thus, once that step is done, your source has been turned into a program that is guaranteed to work.
2. Jq uses the same term.